packages/services/src/models/notification-scanners/DisputeRuled.js

```diff
@@ -0,0 +1,45 @@
+import NotificationScannerBaseModel from './NotificationScannerBaseModel'
+import Network from '@aragonone/court-backend-server/build/web3/Network'
+
+class DisputeRuled extends NotificationScannerBaseModel {
+  async scan() {
+    let notifications = []
+    const nineDaysBeforeNow = Math.floor((Date.now()-(9*this._DAYS))/1000)
+    const query = `
+    {
+      adjudicationRounds(where: {state: Ended, createdAt_gt: ${nineDaysBeforeNow}}, orderBy: createdAt) {
+        dispute {
+          id
+        }
+        jurors {
+          juror {id}
+        } 
+      }
+    }
+    `
+    const { adjudicationRounds } = await Network.query(query)
+    for (const adjudicationRound of adjudicationRounds) {
+      const { 
+        dispute: { id: disputeId },
+        jurors
+      } = adjudicationRound
+      for (const juror of jurors) {
+        notifications.push({ 
+          address: juror.juror.id,
+          details: {
+            emailTemplateModel: {
+              disputeId,
+              disputeUrl: `${this._CLIENT_URL}disputes/${disputeId}`,
+              disputeResult: 'Allowed'      // how do I get this result? I only see finalRuling which is an integer
```

Comment:
@facuspagnuolo @bingen , not sure how to get this ruled state as a readable word (it is included in email template)

Reply:
We can fetch the [`finalRuling`](https://github.com/aragon/court-subgraph/blob/master/schema.graphql#L70) from the `dispute` object in the query. 

You can follow [this spec](https://github.com/aragon/court-dashboard/blob/94c3d738986903e26b78df9aa8a7f3ff0569c724/src/utils/crvoting-utils.js#L4-L10) to parse the final ruling in the same way we are already doing for the dashboard.

Reply:
added